homework-1/src/templates/tips.html

```diff
@@ -0,0 +1,107 @@
+<div class="tips" id="tips">  
+    <div class="tips__section">
+     <div class="tips__area-logo">
+        <a id="tipsLogo" class="tips__logo-link"><img src="/assets/images/SkoutTips.png" alt="" class="tips__logo" ></a>
+     </div>  
+     <div class="tips__first" id="tipsFirstOne">
+        <p class="tips__first-one-letter">1</p>
```

Comment:
please, read about [css counters](https://developer.mozilla.org/ru/docs/Web/CSS/CSS_Lists_and_Counters/Using_CSS_counters) and move this numbers to pseudo element for parent element.

Reply:
<div class="faqs__quetions" id="faqsQuetions">
                <ul class="faqs__quetion">
                    <li class="faqs__quetion-litter">Q</li>
                    <li class="faqs__quetion-text">What are your system requirements?</li>
                    <li class="faqs__down" id="faqsQuetionDown"><i class="fas fa-chevron-down faqs-color-icon"></i></li>
                    <li class="faqs__up" id="faqsQuetionUp"><i class="fas fa-chevron-up faqs-color-icon"></i></li>
                </ul>
                <ul class="faqs__answer"  id="faqsAnswer">
                    <li class="faqs__answer-litter">A</li>
                    <li class="faqs__answer-text">I love you 1</li>    
                </ul>
            </div>

//----done